src/Hl7.Fhir.Serialization/FhirXmlNode.cs

```diff
@@ -438,10 +438,7 @@ object checkRepresentation(ITypedElement node, IExceptionSource ies, object _)
         private ExceptionNotification buildException(string message) => ExceptionNotification.Error(
                 new StructuralTypeException("Parser: " + message));
 
-        public string Serialize(ISourceNode node)
-        {
-            return node.ToXml();
-        }
+        public string OriginalElement => ((XElement)Current).ToString(SaveOptions.DisableFormatting);
```

Comment:
SourceNode already provides this kind of access to the underlying data, could we not push this down into sourcenode, so it ends up with all the other "source xml info" code?